Grunt-ReadMe.md

```diff
@@ -19,16 +19,6 @@ Note that 'grunt serve' supports live reload, i.e. it will monitor for any chang
 
 Run ```grunt debug``` to debug this App locally. This task uses [Node Inspector](https://github.com/node-inspector/node-inspector) to debug your application, and will open the Debugger at 'application.js' in Google Chrome. See the documentation for [Node Inspector](https://github.com/node-inspector/node-inspector) for more information on how to use Node Inspector.
 
```

Comment:
Because these directories don't exist
https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/blob/master/Gruntfile.js#L102
https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/blob/master/Gruntfile.js#L110
https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/blob/master/Gruntfile.js#L120
https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/blob/master/Gruntfile.js#L133